lib/config.js

```diff
@@ -110,7 +110,11 @@ function loadConfig(filePath) {
             // Make the last element in an array take the highest precedence
             config = configExtends.reduceRight(function (previousValue, parentPath) {
 
-                if (isFilePath(parentPath)) {
+                if (parentPath === "eslint:recommended") {
+                    // Add an explicit substitution for eslint:recommended to conf/eslint.json
+                    // this lets us use the eslint.json file as the recommended rules
+                    parentPath = "./conf/eslint.json";
```

Comment:
Will this relative path work all the time? An example of what I'm thinking:

```sh
$ cd /Users/btmills/myproject
$ cat .eslintrc
{
    "extends": "eslint:recommended"
}
$ eslint test.js
```

Would `./conf/eslint.json` resolve to `/Users/btmills/myproject/conf/eslint.json` there?